test/functional/Java8andUp/src/org/openj9/test/com/ibm/jit/Test_JITHelpers.java

```diff
@@ -53,4 +55,194 @@ public static void test_getSuperclass() {
+				Assert.assertTrue(Arrays.equals(buffer, Arrays.copyOfRange(lowercaseUTF16LittleEndianByte, 0, j * 2)),
+					"UTF16 JITHelper lower case conversion with little endian byte arrays of " + j + " letters failed");
+			}
+        	buffer.getClass();
+		}
+	}
+
+	/**
+	 * @tests com.ibm.jit.JITHelpers#toLowerIntrinsicLatin1(char[], char[], int)
+	 */
+    public static void test_toUpperIntrinsicLatin1() {
+		char[] buffer;
+		char[] source, converted;
+
+		for (int j : edgecaseLengths) {
+			source = Arrays.copyOfRange(lowercaseLatin1Char, 0, (j + 1) / 2);
+			converted = Arrays.copyOfRange(uppercaseLatin1Char, 0, (j + 1) / 2);
+
+			if (j % 2 == 1) {
+				source[source.length - 1] &= 0xff00;
+				converted[source.length - 1] &= 0xff00;
+			}
+
+			buffer = new char[(j + 1) / 2];
+			if (helpers.toUpperIntrinsicLatin1(source, buffer, j)) {
+				Assert.assertTrue(Arrays.equals(buffer, converted),
+					"Latin 1 JITHelper upper case conversion with char arrays of " + j + " letters failed");
+			}
+        	buffer.getClass();
+		}
+	}
+
+	/**
+	 * @tests com.ibm.jit.JITHelpers#toUpperIntrinsicLatin1(byte[], byte[], int)
+	 */
+    public static void test_toUpperIntrinsicLatin1two() {
+		byte[] buffer;
+
+		for (int j : edgecaseLengths) {
+			buffer = new byte[j];
+			if (helpers.toUpperIntrinsicLatin1(Arrays.copyOfRange(lowercaseLatin1Byte, 0, j), buffer, j)) {
+				Assert.assertTrue(Arrays.equals(buffer, Arrays.copyOfRange(uppercaseLatin1Byte, 0, j)),
+					"Latin 1 JITHelper upper case conversion with byte arrays of " + j + " letters failed");
+			}
+        	buffer.getClass();
+		}
+	}
+
+	/**
+	 * @tests com.ibm.jit.JITHelpers#toLowerIntrinsicLatin1(char[], char[], int)
+	 */
+    public static void test_toLowerIntrinsicLatin1() {
+		char[] buffer;
+		char[] source, converted;
+
+		for (int j : edgecaseLengths) {
+			source = Arrays.copyOfRange(uppercaseLatin1Char, 0, (j + 1) / 2);
+			converted = Arrays.copyOfRange(lowercaseLatin1Char, 0, (j + 1) / 2);
+
+			if (j % 2 == 1) {
+				source[source.length - 1] &= 0xff00;
+				converted[source.length - 1] &= 0xff00;
+			}
+
+			buffer = new char[(j + 1) / 2];
+			if (helpers.toLowerIntrinsicLatin1(source, buffer, j)) {
+				Assert.assertTrue(Arrays.equals(buffer, converted),
+					"Latin 1 JITHelper lower case conversion with char arrays of " + j + " letters failed");
+			}
+        	buffer.getClass();
+		}
+	}
+
+	/**
+	 * @tests com.ibm.jit.JITHelpers#toLowerIntrinsicLatin1(byte[], byte[], int)
+	 */
+    public static void test_toLowerIntrinsicLatin1two() {
+		byte[] buffer;
+
+		for (int j : edgecaseLengths) {
+			buffer = new byte[j];
+			if (helpers.toLowerIntrinsicLatin1(Arrays.copyOfRange(uppercaseLatin1Byte, 0, j), buffer, j)) {
+				Assert.assertTrue(Arrays.equals(buffer, Arrays.copyOfRange(lowercaseLatin1Byte, 0, j)),
+					"Latin 1 JITHelper lower case conversion with byte arrays of " + j + " letters failed");
+			}
+        	buffer.getClass();
+		}
+	}
```

Comment:
I don't think so, since the result of that method when compiled is dependent on the machine. There's no way we could really test that it returns the correct result.